src/components/VmDialog/index.js

```diff
@@ -33,6 +33,7 @@ import SelectBox from '../SelectBox'
 import { MAX_VM_MEMORY_FACTOR } from '../../constants/index'
 
 import { msg } from '../../intl'
+import { Fragment } from '../utils'
```

Comment:
Can we use `React.Fragment` instead of creating new component?

Reply:
Unfortunately `(props) => props.children` only works since React 16.0. So I resorted to `<div>` wrapper.